Source/engine/random.cpp

```diff
@@ -51,4 +51,9 @@ int32_t GenerateRnd(int32_t v)
 	return AdvanceRndSeed() % v;
 }
 
+bool flipCoin()
+{
+	return GenerateRnd(2) == 0;
```

Comment:
There are 7 cases of this and 40 cases of the other variant, so change this to the common one and change the 7 other cases to use `!flipCoin()`

```suggestion
	return GenerateRnd(2) != 0;
```

Reply:
Alright just that one line fixed the 5 failing item generation tests :)